index.html

```diff
@@ -3,9 +3,37 @@
 <head>
```

Comment:
Better to have a readme so you add how exactly it works, it seems add and saving new recipe does not work